stm32cube/stm32f3xx/README

```diff
@@ -23,7 +23,7 @@ URL:
    http://www.st.com/en/embedded-software/stm32cubef3.html
 
 commit:
-   version 1.10.0
+   version v1.11.0
```

Comment:
Can we add a real commit number (since we should have one now) ?

Reply:
_do you mean, something like:_

commit: 
`   `version v1.11.0 
`   `8fa3aadf0255818f0ca72ba6a5a6731ef8c585fb

Reply:
yes, could be short version of SHA1 though

Reply:
finally, changed as follows:
Status: 
`   `version v1.11.0
..
Commit: 
`   `8fa3aadf0255818f0ca72ba6a5a6731ef8c585fb